darts/backtesting/backtesting.py

```diff
@@ -399,8 +400,13 @@ def backtest_gridsearch(model_class: type,
     """
 
     if (val_series is not None):
-        raise_if_not(train_series.width == val_series.width, "Training and validation series require the same"
-                     " number of components.", logger)
+        if val_series == 'train':
```

Comment:
I can change val_series to take the train TimeSeries, and perform a check `train_series == val_series` to activate this option so there is no abuse

Reply:
If we do that the type will be OK, but I think it might be misleading because the behavior will be different, as we wouldn't in this case evaluate `model.predict(len(train_series))`. I almost prefer taking the "train" string in this case. @pennfranc @guillaumeraille any good idea?

Reply:
I think the easiest and clearest would be to add a separate boolean argument. On top of what Julien mentionned `val_series` ends with series which makes it very confusing to be a string here in my opinion :)

Reply:
Maybe a boolean argument `use_fitted_values` could do indeed. It would override the `val_series` when set.